src/amo/sagas/addonsByAuthors.js

```diff
@@ -1,9 +1,10 @@
 import { call, put, select, takeEvery } from 'redux-saga/effects';
-import { SEARCH_SORT_TRENDING } from 'core/constants';
+import { ADDON_TYPE_THEME, SEARCH_SORT_TRENDING } from 'core/constants';
 import {
+  EXTENSIONS_BY_AUTHORS_PAGE_SIZE,
   FETCH_ADDONS_BY_AUTHORS,
-  ADDONS_BY_AUTHORS_PAGE_SIZE,
   loadAddonsByAuthors,
+  THEMES_BY_AUTHORS_PAGE_SIZE,
```

Comment:
We usually put the constants first, then the rest and both are alphabetically sorted. Sorry for this..